Extension/YetAnotherPartyOrganiser.csproj

```diff
@@ -1,357 +1,357 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What changed here?

Reply:
Think rider and VS save csproj files with different indentation. This'll keep happening unless we can define the indent width in an editorconfig file which works with VS and Rider natively